File: Backend/backend-system-service/Models/AddBuildingRequest.cs
namespace backend_system_service.Models;

public class AddBuildingRequest
{
    public string BuildingName { get; set; }
    public string BuildingDescription { get; set; } 
    public string Street { get; set; }
    public string City { get; set; }
    public string ZipCode { get; set; }
    public string Country { get; set; }
}

File: Flutter-App/app/lib/api/models/requests/add_building_request.dart
class AddBuildingRequest {
  String? name;
  String? description;
  String? street;
  String? city;
  String? zipCode;
  String? country;

  AddBuildingRequest();


  Map<String, dynamic> toJson() {
    final Map<String, dynamic> data = <String, dynamic>{};
    data['buildingName'] = name;
    data['buildingDescription'] = description;
    data['street'] = street;
    data['city'] = city;
    data['zipCode'] = zipCode;
    data['country'] = country;
    return data;
  }
}
